database/migrations/2018_12_17_183200_create_boards_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateBoardsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('boards', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('title', 150);
+            $table->unsignedInteger('user_id');
+            $table->foreign('user_id')->references('id')->on('users')->onDelete('cascade');
+            $table->timestamps();
+            $table->softDeletes();
```

Comment:
Do you plan on using soft deletes? If you do, you have to be consistent in every migration.

Reply:
Yes, so I should add soft deletes to the ideas table as well?

Reply:
Yeah